Sources/Xcodeproj/XcodeProjectModelSerialization.swift

```diff
@@ -574,6 +574,34 @@ public enum PropertyList {
     case string(String)
     case array([PropertyList])
     case dictionary([String: PropertyList])
+
+    static func + (lhs: PropertyList, rhs: PropertyList) -> PropertyList {
```

Comment:
This is no longer required, right?

Reply:
no longer. I still think the serialization FIXME should be addressed, but agree it's not necessarily with this PR